fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -121,8 +121,43 @@ export default Ember.Route.extend(NeedsExistingManifestHelpers, {
       },
       data: {}
     }).then(response => {
-      controller.set('validationErrors', response.validation.errors);
-      controller.set('validationWarnings', response.validation.warnings);
+       if (Ember.isPresent(response.validation)) {
+         controller.set('validationErrors', response.validation.errors);
```

Comment:
I was getting an error here. Are we sure that `response.validation` is returned on success?